README.md

```diff
@@ -48,18 +48,40 @@ up to resemble Vim's edit mode.
 The following keys are currently supported:
 
 - Movement
-  + h: move to the left
-  + j: move down
-  + k: move up
-  + l: move left 
+  + `h`: move to the left
+  + `j`: move down
+  + `k`: move up
+  + `l`: move left
 
 - Quick Movement
-  + Ctrl + F: Page down
-  + Ctrl + B: Page up (not working as well for some reason)
+  + `Ctrl + F`: Page down
+  + `Ctrl + B`: Page up (not working as well for some reason)
+
+- Highlight mode
+  + Press `,` to highlight the current line for easier horizontal reading.
+  + Scrolling up and down will move the highlight bar within the window
+  + Press `,` again to exit highlight mode
+
+- Goto line
+  + Press `;`, then type a line number (e.g. `:1000`) and press `Enter`
+
+- Filter columns
+  + Press `.`, then type a comma-separated list of columns (e.g. `.name, age, survived`) and press `Enter`
+
```

Comment:
I'm curious what the limitations are. Can I do a self join? A having clause? Ranking and partitioning?

Reply:
Definitely yes to self-join and HAVING. Not sure about the others